html/pages/deluser.inc.php

```diff
@@ -14,7 +14,7 @@
             $delete_username = dbFetchCell('SELECT username FROM users WHERE user_id = ?', array($vars['id']));
 
             if ($vars['confirm'] == 'yes') {
-                if (deluser($delete_username)) {
+                if (deluser($vars['id']) >= 0) {
```

Comment:
That function is in all of the authentication modules (well 3/4) so this seemed a bit more central.